src/title.hpp

```diff
@@ -0,0 +1,179 @@
+#pragma once
+
+#include <string>
+#include <iostream>
+
+#include "utils.hpp"
+
+
+//concatenates all strings in vector into a single string
+std::string multiple_lines_into_one(std::vector<std::string>& lines) {
+    std::string title = lines[0];
+    for (size_t i = 1; i < lines.size(); i++) {
+        append_line(title, lines[i]);
```

Comment:
Assuming `lines` is not empty is dangerous, for example [this file](https://github.com/bigno78/pa193-project/files/6506919/261.txt) causes a SIGSEGV.
```suggestion
    std::string title;
    for (const auto & line : lines) {
        append_line(title, line);
```